src/controls/create.ts

```diff
@@ -152,10 +215,10 @@ const create = async () => {
       data: promptAnswers,
     });
 
-    if (response.message === "OpenAI provider key is required") {
+    if (response.message.includes("key is required")) {
```

Comment:
This needs to check if response.message is undefined

```typescript
if(response.message && response.message.includes("key is required"))
```
otherwise it throws an error